### PR TITLE
Use django-celery email as default email backend #271

### LINKED ISSUE #271: [change] Use django-celery email as default email backend
_Continuing from the discussion in https://github.com/openwisp/ansible-openwisp2/pull/262#discussion_r628956198_

Changes needed to be introduced:

1. Add a dedicated variable for setting email backend, see https://github.com/openwisp/ansible-openwisp2/commit/59796a1e231670bffeb084e5a890fda4fc64a3b8
2. Install django-celery-email package by default

**Open for discussion:** Check whether we can get any advantage from adding a dedicated celery queue for emails.

### PR DESCRIPTION
**Pull request by c4llous**
Closes #271

### CODE REVIEW

File: templates/openwisp2/settings.py [hunk 1]
```diff
@@ -479,7 +479,7 @@
 }
 
 INSTALLED_APPS.append('djcelery_email')
-EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
+EMAIL_BACKEND = '{{ openwisp2_email_backend }}'
 {% endif %}
```
Comment (pandafy) [inline templates/openwisp2/settings.py:483]:
```suggestion
{% endif %}

EMAIL_BACKEND = '{{ openwisp2_email_backend }}'
```

The email backend should not depend on availability of the monitoring module

**Review by pandafy — CHANGES_REQUESTED** [review]
Good start @c4llous. Can you also add `openwisp2_email_backend` to the [Role variables section of the README](https://github.com/openwisp/ansible-openwisp2#role-variables)?  In the short description of the role variable also add link to the [django-celery backend](https://github.com/pmclanahan/django-celery-email) package. 

We also need to add a task that installs `django-celery-backend` in https://github.com/openwisp/ansible-openwisp2/blob/master/tasks/pip.yml. The package should be only installed when `openwisp2_email_backend == "djcelery_email.backends.CeleryEmailBackend"`.

**Review by pandafy — CHANGES_REQUESTED** [review]
@c4llous the commit message is not correct, read [OpenWISP's contribution guidelines](https://openwisp.io/docs/developer/contributing.html)

**Review by nemesisdesign — APPROVED** [review]
Thanks @c4llous @pandafy :+1:

**Pull request merged**